### ISSUE TITLE
NPC manual damage roll and saving throw not working.[BUG]

### ISSUE THREAD

**Issue by Vallant85** [MangoFVTT/fvtt-ready-set-roll-5e#77]
**Description**
NPC manual roll damage button does not work. It does nothing. As far as I can tell, it works for player characters. Also the saving throws do not work. Press the button and nothing.
 
**Steps to Reproduce**
1. Go to 'A monster token with manual damage roll enabled'
2. Click on 'Any attack'
3. Scroll down to 'Roll Damage'

**Versions**
 - Module: [1.3.7]
 - dnd5e: [2.0.3]
 - Foundry: [10.286]

**Other Modules**
A list of any other installed modules that might be related to the issue.

**Comment by cs96and** [comment]
I'm also seeing this.  This is the error I get from the SRD Tiger's Bite attack...
```
Uncaught (in promise) TypeError: item.flags[(intermediate value)] is undefined
[Detected 3 packages: ready-set-roll-5e, system:dnd5e, lib-wrapper]
    getFlagValueFromItem item.js:153
    ensureItemParams item.js:194
    getSpecificFieldsFromItem item.js:97
    upgradeToDamageRoll quickroll.js:277
    _processDamageButtonEvent quickcard.js:176
    _setupActionButtons quickcard.js:94
    jQuery 9
    _setupActionButtons quickcard.js:93
    updateBinding quickcard.js:38
    QuickCard quickcard.js:20
    bindChatCard chat.js:30
    registerChatHooks hooks.js:101
    #call foundry.js:724
    call foundry.js:706
    getHTML foundry.js:18033
    postOne foundry.js:73461
    _onCreate foundry.js:18111
    callback foundry.js:12510
    documents foundry.js:12460
    _handleCreateDocuments foundry.js:12460
    _createDocuments foundry.js:12368
    create commons.js:6708
    createDocuments commons.js:6077
    create commons.js:6200
    toMessage quickroll.js:179
    _getItemRoll roll.js:426
    rollItem roll.js:189
    registerItemHooks hooks.js:91
    #call foundry.js:724
    callAll foundry.js:681
    use item.mjs:845
    call_wrapped libWrapper-wrapper.js:507
    rollItemWrapper roll.js:102
    _itemProcessWrapper patching.js:146
    _itemUse patching.js:108
    call_wrapper libWrapper-wrapper.js:616
    use#0 libWrapper-wrapper.js:189
    _onItemUse base-sheet.mjs:994
    activateListeners base-sheet.mjs:655
    jQuery 5
item.js:153:13
```

**Comment by MangoFVTT** [comment]
I believe I have caught the reason for this issue. It has been fixed and should be resolved in the next release.

**Comment by MangoFVTT** [comment]
This issue has now been fixed with Release 1.3.8
